core/src/main/java/nl/nn/adapterframework/senders/Samba2Sender.java

```diff
@@ -65,4 +65,9 @@ public void setRealm(String realm) {
 		getFileSystem().setRealm(realm);
 	}
 
+	@IbisDoc({ "9", "port", "" })
+	public void setPort(Integer port) {
+		getFileSystem().setPort(port);
+	}
```

Comment:
Please move this code to the same relative position as it is in Samba2FileSystem.
Please also replace the IbisDoc in this class with IbisDocRef. For an example on how to do that, see XsltPipe/XsltSender.

Please do the same for Samba2Pipe and Samba2Listener